cql3/statements/create_keyspace_statement.hh

```diff
@@ -78,6 +78,13 @@ public:
     execute(query_processor& qp, service::query_state& state, const query_options& options, std::optional<service::group0_guard> guard) const override;
 
     lw_shared_ptr<data_dictionary::keyspace_metadata> get_keyspace_metadata(const locator::token_metadata& tm, const gms::feature_service& feat);
+
+    virtual ::shared_ptr<event_t> created_event() const override {
+        return make_shared<event_t>(
+                event_t::change_type::CREATED,
+                event_t::target_type::KEYSPACE,
+                keyspace());
+    }
```

Comment:
Please deinline, let's keep the headers clean when there's no reason to inline stuff.

Reply:
ok